docs/src/content/docs/zh-cn/guides/components.mdx

```diff
@@ -237,7 +237,7 @@ import { LinkCard } from '@astrojs/starlight/components';
 
 import { LinkButton } from '@astrojs/starlight/components';
 
-<LinkButton href="/getting-started/">Get started</LinkButton>
+<LinkButton href="/zh-cn/getting-started/">Get started</LinkButton>
 <LinkButton href="https://docs.astro.build" variant="secondary" icon="external">
 	Related: Astro
 </LinkButton>
```

Comment:
@liruifengv Looks like the texts ("Get started" & "Related: Astro") of the `<LinkButton>` example were forgotten during the translation, would you be up to fix them? You can do that in this PR or just suggest the changes in this PR if you prefer.

Reply:
Sure!

Reply:
Amazing, thanks a lot, super appreciated 🙌